lib/active_shipping/package.rb

```diff
@@ -13,19 +13,16 @@ def initialize(grams_or_ounces, dimensions, options = {})
 
       @dimensions = [dimensions].flatten.reject(&:nil?)
 
-      imperial = (options[:units] == :imperial) ||
-                 ([grams_or_ounces, *dimensions].all? { |m| m.respond_to?(:unit) && m.unit.to_sym == :imperial })
```

Comment:
All these checks for `:unit` have been obsolete since [we remove quantified](https://github.com/Shopify/active_shipping/pull/450).